### PR TITLE
[W4.11e][T10-2]Haw Zhe Hao Elroy

### PR DESCRIPTION
**Pull request by elroyhaw**
Add test for isAnyNull

### CODE REVIEW

File: test/java/seedu/addressbook/common/UtilsTest.java [hunk 1]
```diff
@@ -43,4 +43,9 @@ private void assertAreUnique(Object... objects) {
     private void assertNotUnique(Object... objects) {
         assertFalse(Utils.elementsAreUnique(Arrays.asList(objects)));
     }
+
+    @Test
+    public void isAnyNull() {
+        assertTrue(Utils.isAnyNull((Object) null));
```
Comment (yunpengn) [inline test/java/seedu/addressbook/common/UtilsTest.java:49]: You can try different kinds of input supplied to `isAnyNull`. Try to test out all the corner cases as well.

**Review by yunpengn — COMMENTED** [review]
Please close this PR after viewing my comments.
